### PR TITLE
[Feature] Loading spinner

### PR DESCRIPTION
**Pull request by valger11**
## Description
This PR introduces new LoadingSpinner component to the library. It is used to indicate users that application is loading data.

## Motivation and Context
There was need to implement this component as it was designed. And there was need inform users, about loading data with this component.

## How Has This Been Tested?
With Style Guidist On Mac with Chrome and Safari.
## Screenshots (if appropriate):
Screenshot 2022-03-29 at 10 51 18

## Release notes
* Introduces LoadingSpinner component

### CODE REVIEW

File: .styleguidist/styleguidist.sections.js [hunk 1]
```diff
@@ -146,6 +146,12 @@ module.exports = {
                 'SingleSelect/SingleSelect',
               ]),
             },
+            {
+              name: 'LoadingSpinner',
```
Comment (LJKaski) [inline .styleguidist/styleguidist.sections.js:150]: What should the official name for the component be? Was it decided on somewhere? In the designs it's Preloader.

File: src/core/LoadingSpinner/LoadingSpinner.baseStyles.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      margin-top: 6px;
+    }
+    &.fi-loadingSpinner-labelAlign-right {
+      display: flex;
+      align-items: center;
+
+      & .fi-loadingSpinner-label {
+        margin-left: 10px;
+        margin-top: 0;
+        line-height: 1.3em;
```
Comment (LJKaski) [inline src/core/LoadingSpinner/LoadingSpinner.baseStyles.tsx:32]: In the designs the line height is `1.5em`. Where does this value come from?

File: src/core/LoadingSpinner/LoadingSpinner.tsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+              d="M20,0 L20,3 C10.6111593,3 3,10.6111593 3,20 C3,29.3888407 10.6111593,37 20,37 
+              C29.280923,37 36.824796,29.5628044 36.9969921,20.3230397 L37,20 L40,20 C40,31.045695 
+              31.045695,40 20,40 C8.954305,40 0,31.045695 0,20 C0,9.06936433 8.76872859,0.186774951 
+              19.6555106,0.00290705581 L20,0 Z"
+              id="Path"
+              fill="#00347A"
+              fillRule="nonzero"
+            />
+          </g>
+        </svg>
```
[the remark this answers is not in the document]
Comment (ketsappi) [reply src/core/LoadingSpinner/LoadingSpinner.tsx:70]: Good point. We should create the icon in the separate icon repo like with other components.

**Comment by Riippi** [comment]
Continued and replaced by https://github.com/vrk-kpa/suomifi-ui-components/pull/607

**Pull request closed without merging**